stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -41,7 +41,122 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
-  header "SwiftGlibc.h"
+% if CMAKE_SDK not in ["ANDROID"]:
+  header "stdc-predef.h"
+  header "features.h"
+% end
+
+  // C standard library
+  header "complex.h"
+  header "ctype.h"
+  header "errno.h"
+  header "fenv.h"
+  header "float.h"
+  header "inttypes.h"
+  header "iso646.h"
+% if CMAKE_SDK not in ["LINUX", "ANDROID"]:
+  header "libutil.h"
```

Comment:
I checked all these headers in glibc too, and both Bionic and glibc don't have this. I only see a `bsd/libutil.h` from the `libbsd-dev` Ubuntu package, which I don't think the compiler finds.

If this header is wanted on linux, I can change this to `bsd/libutil.h` instead, so that the compiler will find it.